src/worker/jobs/downloads/process_log.rs

```diff
@@ -343,16 +343,16 @@ impl Debug for NameAndVersion {
 
 /// Checks if the given log file has already been processed.
 ///
-/// Calls [`spawn_blocking()`] and acquires a connection from the pool before
+/// Acquires a connection from the pool before
 /// passing it to the [`already_processed_inner()`] function.
```

Comment:
Rewrap:

```suggestion
/// Acquires a connection from the pool before passing it to the [`already_processed_inner()`]
/// function.
```

Reply:
yeah, I mainly kept it like this to reduce the diff size :D